HeavyIonsAnalysis/JetAnalysis/python/rechitanalyzer_cfi.py

```diff
@@ -39,10 +43,13 @@
 
 rechitanalyzerpp = rechitanalyzer.clone(
     vtxSrc = cms.untracked.InputTag("offlinePrimaryVerticesWithBS"),
-    JetSrc = cms.untracked.InputTag("ak3CaloJets")
+    JetSrc = cms.untracked.InputTag("ak4CaloJets"),
+    doHF = cms.untracked.bool(False)
     )
 
 pfTowerspp = pfTowers.clone(
     vtxSrc = cms.untracked.InputTag("offlinePrimaryVerticesWithBS"),
-    JetSrc = cms.untracked.InputTag("ak3CaloJets")
+    JetSrc = cms.untracked.InputTag("ak4CaloJets"),
+    doHF = cms.untracked.bool(False),
+    doTowers = cms.untracked.bool(False)
```

Comment:
please also undo this line, let's keep the default for `doTowers` as true. it can be modified in the config if it is really needed.